backend/user_types.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel
+from typing import Optional, List
+
+
+class UserDetails(BaseModel):
```

Comment:
I think we should add therapist id or something so we can associate with patients and that's how we Will know all the therapist's patients. Or on the other hand keep for each therapist list with all his patient's ids

Reply:
I agree, but it can be added later on